about.html

```diff
@@ -32,17 +32,23 @@
             </div>
         </div>
     </nav>
-    <section id="hero">
-        <div class="right-section">
+
+    <section>
+        <div class="about">
             <h1>About</h1>
-            <p class="subtitle">Typing Test is an open source project created by Arpit Ghura. The purpose of this community project is to collaborate with web developers from around the world. Our goal is to build a useful tool to practice and improve typing skills. If you would like to contribute to our project, click on the GitHub link at the top right of the page.</p>
+            <div class="subtitle">
+            <p> Typing Test is an open source project created by Arpit Ghura.</p>
+            <p> The purpose of this community project is to collaborate with web developers from around the world.</p>
```

Comment:
I think you should not break this text into multiple paragraphs.

Reply:
ok, I will make these changes.